cmd/gobrew/main.go

```diff
@@ -2,12 +2,13 @@ package main
 
 import (
 	"flag"
-	"log"
```

Comment:
gets rid of `log` all together. Leaving now two output formats `fmt` and `color`.
Ref #87